src/index.html

```diff
@@ -5,18 +5,81 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <link rel="stylesheet" href="node_modules/uikit/dist/css/uikit.min.css">
-    <title>Apka</title>
+    
+    <title>Practice Control</title>
 </head>
 <body>
-  
-    <div class="uk-container-fluid">
-        <div class="uk-card uk-card-body uk-card-primary">
-            <h3 class="uk-card-title">Example headline</h3>
+    <!-- login form -->
+    <div class="container-login">
+    <div class="wrap-login">
+    <form id="login_form">
+        <div class="uk-margin">
+            <div class="uk-inline">
+                <span class="uk-form-icon" uk-icon="icon: user"></span>
+                <input id='email_field' class="uk-input uk-close" type="text" placeholder="E-mail">
+            </div>
+        </div>
+    
+        <div class="uk-margin">
+            <div class="uk-inline">
+                <span class="uk-form-icon" uk-icon="icon: lock"></span>
+                <input id='password_field' class="uk-input uk-close" type="password" placeholder="Password">
+            </div>
+        </div>
+        
+        <div class="uk-margin">
+            <button id='login_field' class="uk-button uk-button-primary uk-width-1-1" type="button" data-uk-button>Login</button>
+        </div>
 
-            <button class="uk-button uk-button-default" uk-tooltip="title: Hello World">Hover</button>
+        <div class="text-center">
+            <span class="txt1">
+                Don’t have an account?
+            </span>
+            <a class="txt2" href="#register_form" >
+                Sign Up
+            </a>
         </div>
+    </form>
+    <div id="user_div" class="loggedin-div">
+        <h3>Welcome User</h3>
+        <p id="user_para"> Welcome to firebase web login exampe. u are logged in!</p>
+        <button id='logout_field' class="uk-button uk-button-primary uk-width-1-1" type="button" data-uk-button>Wyloguj</button>
+        <button id='verification_field' class="uk-button uk-margin uk-button-primary uk-width-1-1" type="button" data-uk-button>Send verification mail</button>
     </div>
+</div>
+</div>
+    <!-- //register form -->
+    <div class="container-login">
```

Comment:
Coś to niepoprawnie znika czy pojawia się :p, trzeba będzie pomyśleć jak to rozwiązać

Reply:
Bo form jest w wrapperze glownym dlatego nie znika calkiem, to tylko taka testowa implementacja ez do fixniecia jak bedziemy wiedziec jak podzielic te podstrony, ale mowiles o jquery mobile wiec to zalatwi sprawe :))